components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -2092,16 +2054,44 @@ void AdsServiceImpl::GetAdConversions(
           std::move(callback)));
 }
 
-void AdsServiceImpl::EventLog(
-    const std::string& json) const {
-  VLOG(0) << "AdsService Event Log: " << json;
+void AdsServiceImpl::Log(
+    const char* file,
+    const int line,
+    const ads::LogSeverity severity,
+    const std::string& message) const {
+  ::logging::LogSeverity log_severity;
+
+  switch (severity) {
+    case ads::LogSeverity::LOG_INFO: {
+      log_severity = ::logging::LOG_INFO;
+      break;
+    }
+
+    case ads::LogSeverity::LOG_WARNING: {
+      log_severity = ::logging::LOG_WARNING;
+      break;
+    }
+
+    case ads::LogSeverity::LOG_ERROR: {
+      log_severity = ::logging::LOG_ERROR;
+      break;
+    }
+  }
+
+  if (::logging::ShouldCreateLogMessage(log_severity)) {
+    ::logging::LogMessage(file, line, log_severity).stream() << message;
+  }
 }
 
-std::unique_ptr<ads::LogStream> AdsServiceImpl::Log(
+void AdsServiceImpl::VerboseLog(
     const char* file,
     const int line,
-    const ads::LogLevel log_level) const {
-  return std::make_unique<LogStreamImpl>(file, line, log_level);
+    const int verbose_level,
+    const std::string& message) const {
+  const int vlog_level = ::logging::GetVlogLevelHelper(file, strlen(file));
+  if (verbose_level <= vlog_level) {
+    ::logging::LogMessage(file, line, -verbose_level).stream() << message;
```

Comment:
Is the minus symbol intentional there? Assuming so, but wanted to check.

Reply:
The minus is intentional as this is how Chromium logging is aware of VLOG (negative values) or LOG (positive values)